<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>974</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: URL(:&quot;/icone/iconeAppli.png&quot;);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"> background-image: url(:/backFINAL.png);</string>
   </property>
   <widget class="QPushButton" name="BouttonJouer">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>560</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-image: url(:/pret.png);
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="dessin">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>630</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(&quot;none&quot;);
color: rgb(255, 203, 0);
background-color:transparent;
border:none;
font: 75 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Comic Sans MS'; font-size:14pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1151</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QGroupBox" name="horizontalGroupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>690</y>
      <width>961</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-image: url(:/none.png);
border:none;</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="log">
         <property name="styleSheet">
          <string notr="true">background-color:transparent;
font: 75 12pt &quot;Comic Sans MS&quot;;
border:none;</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Comic Sans MS'; font-size:12pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="logEnvoyer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:transparent;
color: rgb(251, 28, 111);
padding-top:20px;
font: 75 14pt &quot;Comic Sans MS&quot;;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BouttonEnvoyer">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="GbListeJoueurs">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>680</y>
      <width>191</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-image: url(:/none.png);
border: none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="logListeJoueur">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>161</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color: rgb(84, 175, 255);
font: 75 14pt &quot;Comic Sans MS&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>551</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-image: url(:/none.png);
border:none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSlider" name="sliderLargeur">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelLargeur">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(&quot;none&quot;);
color: rgb(255, 255, 255);
background-color:transparent;
border:none;
font: 75 14pt &quot;Comic Sans MS&quot;;</string>
     </property>
     <property name="text">
      <string>Largeur du stylo : 5</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCouleur">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="boutonPalette">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>91</width>
       <height>101</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-image: url(:/palette.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>sliderLargeur</zorder>
    <zorder>labelLargeur</zorder>
    <zorder>boutonPalette</zorder>
    <zorder>labelCouleur</zorder>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>610</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-image: url(:/none.png);
font: 75 12pt &quot;Comic Sans MS&quot;;
color: rgb(255, 255, 255);
border:none;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="GbConnexion">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Connexion</string>
    </property>
    <widget class="QPushButton" name="ButtonConnexion">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Connexion</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pseudo : </string>
     </property>
    </widget>
    <widget class="QLabel" name="labIp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse IP :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pseudo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;Comic Sans MS&quot;</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="adresseIp">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;Comic Sans MS&quot;</string>
     </property>
     <property name="text">
      <string>10.1.48.240</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>600</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(&quot;none&quot;);
color: rgb(255, 255, 255);
background-color:transparent;
border:none;
font: 75 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Mot à dessiner : </string>
    </property>
   </widget>
   <zorder>BouttonJouer</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>horizontalGroupBox_2</zorder>
   <zorder>GbListeJoueurs</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>GbConnexion</zorder>
   <zorder>label</zorder>
   <zorder>dessin</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1138</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pseudo</tabstop>
  <tabstop>adresseIp</tabstop>
  <tabstop>ButtonConnexion</tabstop>
  <tabstop>BouttonJouer</tabstop>
  <tabstop>logEnvoyer</tabstop>
  <tabstop>BouttonEnvoyer</tabstop>
  <tabstop>boutonPalette</tabstop>
  <tabstop>sliderLargeur</tabstop>
  <tabstop>dessin</tabstop>
  <tabstop>log</tabstop>
  <tabstop>logListeJoueur</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
